Makefile

```diff
@@ -7,6 +7,11 @@ PEM := $(PEM_DIR)/cert.pem
 NAMESERVER := 8.8.8.8
 RESOLV_CONF_DIR := endpoint/endpointComp
 OUTPUT_BASE_DIR := output_base
+# Endpoint build target, the localhost is the platform intended to your development system.
```

Comment:
`Endpoint build target. The localhost means the platform of your development system.`